robocop/run.py

```diff
@@ -5,14 +5,15 @@
 from robocop import checkers
 from robocop.config import Config
 from robocop import reports
-from robocop.utils import DisablersFinder
+from robocop.utils import DisablersFinder, FileType, FileTypeChecker
 
 
 SUPPORTED_FORMATS = ('.robot')
 
 
 class Robocop:
     def __init__(self):
+        self.files = dict()
```

Comment:
Why not `self.files = {}`?

Reply:
You're right and I'm not consistent through the same class. I will change this to {}